test/cli/utils_test.go

```diff
@@ -271,7 +274,137 @@ func getSyftBinaryLocation(t testing.TB) string {
 		// SYFT_BINARY_LOCATION is the absolute path to the snapshot binary
 		return os.Getenv("SYFT_BINARY_LOCATION")
 	}
-	return getSyftBinaryLocationByOS(t, runtime.GOOS)
+	bin := getSyftBinaryLocationByOS(t, runtime.GOOS)
+
+	// only run on valid bin target, when not running in CI
+	if bin != "" && os.Getenv("CI") == "" {
```

Comment:
I'm unsure the best way to determine if this is being run locally. It would also be fine (for me, at least) if this only runs when running a single test.